elasticdl/python/master/master.py

```diff
@@ -184,10 +189,6 @@ def run(self):
                         self.worker_manager.update_status(
                             WorkerManagerStatus.FINISHED
                         )
-                    if self.checkpoint_output_path:
```

Comment:
So we save the saved model but nothing else, e.g. checkpoint?

Reply:
The checkpoint is saved regularly. At the last step, we export to the SavedModel as the final format.